package.json

```diff
@@ -15,12 +15,14 @@
   },
   "keywords": [],
   "dependencies": {
-    "fs-blob-store": "^5.2.1",
+    "@digidem/atomic-fs-blob-store": "^5.3.0",
```

Comment:
We are Using that fork in order to use a fork of fs-create-write-stream-atomic which adds a new option to allow a custom naming function for the temp files. The opts are passed through by atomic-FS-blob-Store so if I get a PR accepted as a minor release it would just work. 
We need the custom naming function so that the list function works (so it doesn’t list temp files left by failed rights).

Reply:
sioundssounds good